dashboard/app/controllers/xhr_proxy_controller.rb

```diff
@@ -63,6 +63,7 @@ class XhrProxyController < ApplicationController
     api.zippopotam.us
     bible-api.com
     code.org
+    codeorg-api.com
```

Comment:
we should add a comment (maybe with a link to the documentation) to make it clear that this API isn't owned/maintained by code.org. ruby doesn't support comments in a `%w()` block like this, so the comment will have to go at the top (where the other 'code.org' comment is), unfortunately

Reply:
Great callout Maddie - I'll add that comment